bin/utils.py

```diff
@@ -0,0 +1,26 @@
+import xlsxwriter
```

Comment:
Because of this, `README.md` has to be updated to include `pip install xlsxwriter`

Reply:
done